sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.controller.validateOffering.e2e-spec.ts

```diff
@@ -231,6 +201,440 @@ describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering",
+    const payload = {
+      offeringName: "Offering validation",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.partTime,
+      offeringDelivered: OfferingDeliveryOptions.Onsite,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2024-07-01",
+      studyEndDate: "2024-08-16",
+      lacksStudyBreaks: false,
+      studyBreaks: [
+        {
+          breakStartDate: "2024-08-05",
+          breakEndDate: "2024-08-16",
+        },
+      ],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 1234,
+      programRelatedCosts: 3211,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+      courseLoad: 30,
+    };
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        offeringStatus: "Approved",
+        errors: [],
+        infos: [
+          {
+            typeCode:
+              OfferingValidationInfos.InvalidStudyBreaksCombinedThresholdPercentage,
+            message:
+              "The combined study breaks exceed the 10% threshold as outlined in StudentAid BC policy.",
+          },
+        ],
+        warnings: [],
+        studyPeriodBreakdown: {
+          fundedStudyPeriodDays: 39.7,
+          totalDays: 47,
+          totalFundedWeeks: 6,
+          unfundedStudyPeriodDays: 7.3,
+        },
+      });
+  });
+
+  it("Should return error and warning when data with total funded weeks less than minimum allowed weeks and study breaks is passed in for a part-time education program.", async () => {
+    // Arrange
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    const endpoint = `/institutions/education-program-offering/location/${collegeFLocation.id}/education-program/${collegeFPartTimeProgram.id}/validation`;
+    const payload = {
+      offeringName: "Offering validation",
+      yearOfStudy: 1,
+      offeringIntensity: OfferingIntensity.partTime,
+      offeringDelivered: OfferingDeliveryOptions.Onsite,
+      hasOfferingWILComponent: "no",
+      studyStartDate: "2024-07-01",
+      studyEndDate: "2024-08-16",
+      lacksStudyBreaks: false,
+      studyBreaks: [
+        {
+          breakStartDate: "2024-07-29",
+          breakEndDate: "2024-08-16",
+        },
+      ],
+      offeringType: OfferingTypes.Public,
+      offeringDeclaration: true,
+      actualTuitionCosts: 1234,
+      programRelatedCosts: 3211,
+      mandatoryFees: 456,
+      exceptionalExpenses: 555,
+      courseLoad: 30,
+    };
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        offeringStatus: "Creation pending",
```

Comment:
I would rather use enum instead of string here. `OfferingStatus.CreationPending`